deckhouse-controller/pkg/controller/module-controllers/release/controller.go

```diff
@@ -150,87 +149,82 @@ func NewModuleReleaseController(
 		Complete(ctr)
 }
 
-func (c *moduleReleaseReconciler) emitRestart(msg string) {
-	c.m.Lock()
-	c.delayTimer.Reset(3 * time.Second)
-	c.restartReason = msg
-	c.m.Unlock()
+func (r *moduleReleaseReconciler) emitRestart(msg string) {
+	r.m.Lock()
+	r.delayTimer.Reset(3 * time.Second)
+	r.restartReason = msg
+	r.m.Unlock()
 }
 
-func (c *moduleReleaseReconciler) restartLoop(ctx context.Context) {
+func (r *moduleReleaseReconciler) restartLoop(ctx context.Context) {
 	for {
-		c.m.Lock()
+		r.m.Lock()
 		select {
-		case <-c.delayTimer.C:
-			if c.restartReason != "" {
-				c.logger.Infof("Restarting Deckhouse because %s", c.restartReason)
+		case <-r.delayTimer.C:
+			if r.restartReason != "" {
+				r.logger.Infof("Restarting Deckhouse because %s", r.restartReason)
 
 				err := syscall.Kill(1, syscall.SIGUSR2)
 				if err != nil {
-					c.logger.Fatalf("Send SIGUSR2 signal failed: %s", err)
+					r.logger.Fatalf("Send SIGUSR2 signal failed: %s", err)
 				}
 			}
-			c.delayTimer.Reset(3 * time.Second)
+			r.delayTimer.Reset(3 * time.Second)
 
 		case <-ctx.Done():
 			return
 		}
 
-		c.m.Unlock()
+		r.m.Unlock()
 	}
 }
 
 // only ModuleRelease with active finalizer can get here, we have to remove the module on filesystem and remove the finalizer
-func (c *moduleReleaseReconciler) deleteReconcile(ctx context.Context, mr *v1alpha1.ModuleRelease) (ctrl.Result, error) {
-	res := ctrl.Result{}
```

Comment:
please, return alias.
it makes code more readeble

Reply:
I don't see the point in creating a separate variable when we can just use [named result](https://go.dev/doc/effective_go#named-results)

Reply:
> I don't see the point in creating a separate variable when we can just use [named result](https://go.dev/doc/effective_go#named-results)

then we need to fix 200 line https://github.com/deckhouse/deckhouse/pull/10105/files#diff-899346a0481b3853b6ef509ea9f5895c648bda8532399ef3938ad02fb22cf008R200